packages/api/tests/user.test.js

```diff
@@ -367,8 +367,8 @@ describe('User Tests', () => {
       });
 
       test('Should post then get a valid user and user spotlight', async (done) => {
-        const fakeUser = mocks.fakeUser();
-
+        jest.setTimeout(90000);
```

Comment:
Why are we modifying the default timeout interval 🤔  [setTimeout](https://jestjs.io/docs/jest-object#jestsettimeouttimeout)?